### ISSUE TITLE
Can't install on OSX 10.9.2

### ISSUE THREAD

**Issue by RachelBunder** [olgabot/prettyplotlib#49]
When I try to install, I get the following error message:

> clang: error: unknown argument: '-mno-fused-madd' [-Wunused-command-line-argument-hard-error-in-future]

This seems to be problem with the recently updated xcode. A temporary solution is given here (http://kaspermunck.github.io/2014/03/fixing-clang-error/). Would it be possible to implement it into prettypot?

**Comment by jankatins** [comment]
This is probably related to matplotlib, can you test to install matplotlib and if that fails report the bug there?

**Comment by RachelBunder** [comment]
Yes, same error when I try to install through matplotlib when using the tarball (I can install it using the dmg no problem). I'll report the problem to matplotlib as well.

Since I already have matplotlib on my system, why does prettyplot need to install it again?

**Comment by olgabot** [comment]
Which version of matplotlib do you have? Prettyplotlib requires matplotlib > 1.2.1 for install. If it's greater than that, then I don't know... maybe some issue with pip?

**Comment by RachelBunder** [comment]
I had matplotlib 1.3.1 installed when I first had this issue. 

In the end, I followed the advice from the people at [matplotlib](https://github.com/matplotlib/matplotlib/issues/2969#issuecomment-40875862) (remove all instances of -mno-fused-madd from System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/_sysconfigdata.py). I then installed matplotlib 1.3.1 from the tarball, then installed prettyplotlib. For some reason now it was satisfied with matplotlib 1.3.1 and didn't attempt to install it again. 

So I got it working :). Thanks everyone for your help.
